test/filesystem-spec.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const assert = require('assert')
+const fs = require('../lib/wrapped-fs')
+const path = require('path')
+const rimraf = require('rimraf')
+
+const Filesystem = require('../lib/filesystem')
+
+describe('filesystem', function () {
+  beforeEach(() => { rimraf.sync(path.join(__dirname, '..', 'tmp'), fs) })
+
+  it('should does not throw error when src path include symbol link', async () => {
+    /* eslint-disable no-irregular-whitespace */
+    /**
+     * Directory structure:
+     * tmp
+     * ├── private
+     * │   └── var
+     * │       ├── app
+     * │       │   └── file.txt -> ../file.txt
+     * │       └── file.txt
+     * └── var -> private/var
+     */
```

Comment:
Seems like the comment uses U+00A0 : NO-BREAK SPACE [NBSP] instead of U+0020 : SPACE [SP]. Replacing the former with the latter seems to fix the ESLint warning altogether.

```suggestion
    /**
     * Directory structure:
     * tmp
     * ├── private
     * │   └── var
     * │       ├── app
     * │       │   └── file.txt -> ../file.txt
     * │       └── file.txt
     * └── var -> private/var
     */
```